packages/marketplace/contracts/exchange/ExchangeCore.sol

```diff
@@ -478,8 +461,9 @@ abstract contract ExchangeCore is Initializable, ContextUpgradeable, TransferExe
             }
         }
 
+        // TODO: can we move this out so we don't need to pass from ?
```

Comment:
No, this event should be emitted in the same function as the change in the variable `fills`